pkg/destroy/openstack/openstack.go

```diff
@@ -651,10 +650,10 @@ func deleteCustomRouterInterfaces(opts *clientconfig.ClientOpts, filter Filter,
 		return true, nil
 	}
 
-	// Only proceed with clean up if is a provided Network
+	// Only proceed with clean up if the network was created by OpenShift
 	networkListOpts := networks.ListOpts{
-		NotTags: strings.Join(tags, ","),
```

Comment:
Removing it will remove the provided Network and Router

Reply:
yeah we don't want that for sure.